src/sync/models/centralizedOracle.js

```diff
@@ -40,6 +37,7 @@ class CentralizedOracle {
       txid: this.txid,
       topicAddress:this.eventAddress,
       resultSetterAddress:this.oracle,
+      resultSetterQAddress: qDecoder.toQtumAddress(this.oracle),
```

Comment:
should you add a var to control `Mainnet` vs `Testnet` state?

Reply:
@dwalintukan probably a more elegant way needed instead of in this scope. So prefer to leave it as it is now